README.md

```diff
@@ -135,6 +140,27 @@ my-teradata-db-profile:
       port: <port>
 ```
 
+### Retries
```

Comment:
This update needs to be published to https://docs.getdbt.com/reference/warehouse-setups/teradata-setup and possibly https://docs.getdbt.com/reference/resource-configs/teradata-configs . This is done by submitting a PR to https://github.com/dbt-labs/docs.getdbt.com . For example, see https://github.com/dbt-labs/docs.getdbt.com/pull/1027

Reply:
Sure, Adam. I will create a branch in https://github.com/dbt-labs/docs.getdbt.com and make the required changes.